readthedocs/templates/projects/project_version_list.html

```diff
@@ -26,6 +26,21 @@
           <h1>{% trans "Active Versions" %}</h1>
         </div>
 
+        {% block resync_versions %}
+          {% if request.user|is_admin:project %}
+            <div class="button-bar">
+              <ul>
+                <li>
+                  <form method="post" action="{% url 'project_version_sync' project_slug=project.slug %}">
+                    {% csrf_token %}
+                    <input type="submit" value="{% trans 'Sync Versions' %}">
```

Comment:
I think a refresh icon button probably communicates what we need here and doesn't require us to add copy to our UI.

Reply:
Okay :)

Reply:
I have added this one:
![Screenshot from 2019-11-29 01-33-27](https://user-images.githubusercontent.com/24854406/69828975-4bfc6980-1248-11ea-8b50-ea10481dc5b1.png)

Another option might be:
![Screenshot from 2019-11-29 01-33-11](https://user-images.githubusercontent.com/24854406/69828976-4d2d9680-1248-11ea-95b8-2b804c406b92.png)